cli/ops/worker_host.rs

```diff
@@ -191,13 +191,11 @@ fn op_create_worker(
   let mut state = state.borrow_mut();
   let global_state = state.global_state.clone();
   let permissions = state.permissions.clone();
-  let referrer = state.main_module.to_string();
   let worker_id = state.next_worker_id;
   state.next_worker_id += 1;
   drop(state);
 
-  let module_specifier =
-    ModuleSpecifier::resolve_import(&specifier, &referrer)?;
+  let module_specifier = ModuleSpecifier::resolve_import(&specifier, "")?;
```

Comment:
use `ModuleSpecifier::resolve_url` and revert changes to `core/module_specifier.rs`?